VideoLocker/AndroidManifest.xml

```diff
@@ -225,7 +225,7 @@
         <activity android:name=".profiles.UserProfileActivity"
             android:screenOrientation="portrait"
             android:theme="@style/AppTheme.NoShadow"
-            android:label=""/>
+            android:label="@string/profile_title"/>
```

Comment:
Yeah, that's because we currently use the username as the title (but hide it if the username is already visible on the screen).

To remove that, we'll want to delete these lines from UserProfileFragment:
```java
viewHolder.appbar.addOnOffsetChangedListener(new AppBarLayout.OnOffsetChangedListener() {
            @Override
            public void onOffsetChanged(AppBarLayout appBarLayout, int verticalOffset) {
                if (verticalOffset <= -viewHolder.nameText.getBottom()) {
                    getActivity().setTitle(getUsername());
                } else {
                    getActivity().setTitle("");
                }
            }
        });
```